packages/devextreme/js/__internal/ui/drawer/m_drawer.ts

```diff
@@ -508,6 +508,9 @@ const Drawer = (Widget as any).inherit({
         this._dimensionChanged();
         break;
       case 'opened':
+        if (!this._whenAnimationCompleted) {
```

Comment:
What if it's not a first animation, and this cache already exists but resolved?

Reply:
the solution was reworked